unit-tests/func/test-set-option.py

```diff
@@ -86,48 +86,41 @@ def check_color_frame_drops(frame):
 depth_sensor.set_option( rs.option.visual_preset, int(rs.l500_visual_preset.max_range) )
 
 
-
 #############################################################################################
-# Test #2
-
+def test_option_changes( sensor ):
+    options = sensor.get_supported_options()
+    for option in options:
+        try:
+            if sensor.is_option_read_only(option): 
+                continue
+            old_value = sensor.get_option( option )
+            range = sensor.get_option_range( option )
+            new_value = range.min
+            if old_value == new_value:
+                new_value = range.max
+            if not log.d( str(option), old_value, '->', new_value ):
```

Comment:
What does if not log.d means?
I don't see log.d return a value.

Reply:
log.d should return a value, true if something was written to the log (debug is on)

Reply:
OK, I guess I was looking on an old version of log.py